app/views/projects/index.html.erb

```diff
@@ -7,15 +7,36 @@
 <div id="projects" class="projects-list">
   <div class="row item">
     <div class="col-md-12">
-      <div class="col-md-8">
-        <%= render 'activities_group', activities_group: Activity.grouped_activities(@projects, 1.week.ago) %>
-      </div>
+      <% unless @projects.empty? %>
 
-      <div class="col-md-4">
-        <ul class="list-group">
+        <div class="col-md-6">
+          <% unless Activity.grouped_activities(@projects, 1.week.ago).empty? %>
```

Comment:
What are you think to move this query to instance variable into Project Controller?:
 `@activities_group = Activity.grouped_activities(@projects, 1.week.ago)`